src/components/common/FeatureBox/FeatureBox.module.scss

```diff
@@ -4,6 +4,7 @@
   border: 1px solid $feature-border-color;
   text-align: center;
   margin-top: 40px;
+  min-height: 140px;
```

Comment:
postaraj sie to zrobić bez używania sztywnych wartości wyrażonych w px

Reply:
Zmieniłem wartość na 9rem